pipeline-model-definition/pom.xml

```diff
@@ -120,7 +120,7 @@
     <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>git</artifactId>
-      <scope>test</scope>
+      <optional>true</optional>
```

Comment:
Already tried that - only difference was scope. I'm checking the actual hpi files used in the tests now...